packages/export/src/excel.js

```diff
@@ -0,0 +1,73 @@
+import _ from 'lodash/fp.js'
+import writeXlsxFile from 'write-excel-file/node'
+
+const convertToExcelCell = (value) => {
+  return {
+    type: String,
+    typeof: 'string',
+    wrap: true,
+    value: `${value}`,
+  }
+}
+
+let transformLabels = _.map(_.get('label'))
+
+export default ({
+  stream, // writable stream target stream
+  iterableData, // iterator for each page of an array of objects
+  // order list of which indicates the header label,
+  // display function for the field,
+  // and key of the record.
+  // [{ key: string(supports lodash dot notation), label: string, display: function(value, {key, record, transform})}...]
+  transform,
+  headers = null, // array of strings to use as headers, array or arrays for multi-line headers
+  onWrite = _.noop, // function to intercept writing a page of records
+}) => {
+  const excelData = [
+    _.map(
+      (value) => ({ value, fontWeight: 'bold' }),
+      headers || transformLabels(transform)
+    ),
+  ]
```

Comment:
Will this work if `headers` is an array of arrays?

Reply:
the headers are already being used like this in the CSV file from what I remember, so I did not consider them to be an array of arrays, if that was the case the CSV would not work either from what I read about that library. Can you give an example you have in mind for me to test?

Reply:
I don't have an example I was just going off of the comment for the `headers` argument

> array of strings to use as headers, array or arrays for multi-line headers